bescort.lic

```diff
@@ -1008,6 +1013,35 @@ class Bescort
+  case bput('go barge', "One of the barge's crew members stops you and requests a transportation fee of 65 lirums as you board the craft", 'You hand him', "The barge .* isn't docked, so you're stuck here until the next one arrives", "The barge .* has just pulled away from the dock", 'There is no ferry here to go aboard', 'Come back when you can afford the fare')                                                                                                                                       
+  when 'You hand him', "One of the barge's crew members stops you and requests a transportation fee of 65 lirums as you board the craft"                             
+    waitfor 'reaches its dock and its crew ties the barge off'                                                                                                       
+    if DRRoom.room_objs.find { |x| x =~ /Langenfirth wharf/ }                                                                                                        
+      move 'go wharf'                                                                                                                                                
+    else                                                                                                                                                             
+      move 'go pier'                                                                                                                                                 
+    end                                                                                                                                                              
+  when /The barge .* has just pulled away from the dock/, /The barge .* isn't docked, so you're stuck here until the next one arrives/                               
+    pause 1 while DRRoom.room_objs.find { |x| x =~ /the barge/ }                                                                                                     
+    take_rh_lang_barge(mode)                                                                                                                                         
+  when 'Come back when you can afford the fare'                                                                                                                      
+    echo('Your fare has mysteriously disappeared!')                                                                                                                  
+    return                                                                                                                                                           
+    end                                                                                                                                                                
```

Comment:
2 extra spaces? did you mean to have another end also to end take_rh_lang_barge?